iamctl/pkg/utils/commonUtils.go

```diff
@@ -125,7 +176,12 @@ func SendImportRequest(importFilePath, fileData, resourceType string) error {
 
 func SendUpdateRequest(resourceId, importFilePath, fileData, resourceType string) error {
 
-	reqUrl := SERVER_CONFIGS.ServerUrl + "/t/" + SERVER_CONFIGS.TenantDomain + "/api/server/v1/" + getResourcePath(resourceType) + "/" + resourceId + "/import"
+	var reqUrl string
+	if resourceType == APPLICATIONS {
+		reqUrl = SERVER_CONFIGS.ServerUrl + "/t/" + SERVER_CONFIGS.TenantDomain + "/api/server/v1/" + getResourcePath(resourceType) + "/import"
+	} else {
+		reqUrl = SERVER_CONFIGS.ServerUrl + "/t/" + SERVER_CONFIGS.TenantDomain + "/api/server/v1/" + getResourcePath(resourceType) + "/" + resourceId + "/import"
+	}
```

Comment:
we could think of reusing the buildRequestUrl method here as well

Reply:
Added a new method `buildRequestUrl` that returns the request url for each request.